routes/account.js

```diff
@@ -611,6 +784,14 @@ module.exports = function (
 
   if (isProduction) {
     delete routes[0].config.validate.payload.preVerified
+
+    for (var i = 0; i < routes.length; ++i) {
+      var route = routes[i];
+      if (route.path === '/account/lock') {
+        routes = routes.splice(i, 1)
```

Comment:
Or perhaps we should switch it around to an `isNotProduction` test that adds stuff in rather than taking it away.  But I guess that's not really in the scope of this PR :-)